Сетка страниц и крупных блоков на флексах

alina-kh opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/36/user/2299541/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2299541-gllacy-36/2/

alina-kh committed:
Добавляет сетку страниц на флексах

Komnatnaya commented:
![image](https://user-images.githubusercontent.com/20788716/215296901-893d7d3c-15be-49d8-8c6b-3a65514abfa3.png)
Не нужно фиксировать высоту блоков, чтобы не было выпадения контента, как в данном случае.
Блоки должны растягиваться за счет контента и внутренних отступов, в крайнем случае можно поставить min-height.
Стоит убрать height со всех блоков.

alina-kh committed:
Исправляет замечания по основнойсетке

Убрала фиксированную высоту элементов, прописала размер и повторение бекграунда. Оставила высоту у кнопок которые пока сделаны с бекграундом, что бы их было видно)
